pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1978,6 +1978,6 @@ type RegistryServiceConfig struct {
 	// description: |
 	//   External service endpoint.
 	//   examples:
-	//     - value: 'constants.DefaultDiscoveryServiceEndpoint'
```

Comment:
there _should_ be a way to use the constant here to avoid duplication... probably @Unix4ever knows?

Reply:
I believe dropping the quotes should allow this to be a value directly?

Reply:
I tried that back then; it wasn't enough.

The real problem was with the indentation before `examples`.